client/app/bundles/course/assessment/question/programming/components/OnlineEditorCppView.scss

```diff
@@ -0,0 +1,30 @@
+// scss-lint:disable ImportantRule
+.deleteButtonCell {
+  padding-left: 12px !important;
```

Comment:
Not very sure, but the python css file uses this as well (its just copied over). Just that later when adding a reset button, will likely have to add more css code into this file.